### PR TITLE
PCHR-776 - adding 'cycle_is_active' field for AppraisalCycle entity.

### PR DESCRIPTION
**Pull request by coldrunKacper**
Adding 'cycle_is_active' field for AppraisalCycle entity. The field can take tinyint "0" / "1" value (default is "1").

### CODE REVIEW

File: uk.co.compucorp.civicrm.appraisals/CRM/Appraisals/DAO/AppraisalCycle.php [hunk 1]
```diff
@@ -248,6 +254,18 @@ static function &fields()
             'optionGroupName' => 'appraisal_cycle_type',
           )
         ) ,
+        'cycle_is_active' => array(
+          'name' => 'cycle_is_active',
+          'type' => CRM_Utils_Type::T_INT,
```
Comment (Agares4) [inline uk.co.compucorp.civicrm.appraisals/CRM/Appraisals/DAO/AppraisalCycle.php]: Can't we use CRM_Utils_Type::T_BOOLEAN here?

File: uk.co.compucorp.civicrm.appraisals/CRM/Appraisals/DAO/AppraisalCycle.php [hunk 2]
```diff
@@ -104,6 +104,12 @@ class CRM_Appraisals_DAO_AppraisalCycle extends CRM_Core_DAO
    * @var int
    */
   public $type_id;
+  /**
+   * Is Appraisal Cycle active?
+   *
+   * @var int
```
Comment (Agares4) [inline uk.co.compucorp.civicrm.appraisals/CRM/Appraisals/DAO/AppraisalCycle.php]: Why not `@var boolean`?

**Comment by coldrunKacper** [comment]
Right, sounds reasonable. Changed.

**Comment by Agares4** [comment]
@coldrunKacper please add JIRA tasks' id's to the commit messages. And squash the last two if possible

**Comment by Agares4** [comment]
Approved

**Pull request merged**